src/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -872,28 +895,16 @@ public static bool IsDefined(Type enumType, object value)
 
         public static string Format(Type enumType, object value, string format)
         {
-            if (enumType == null)
-                throw new ArgumentNullException(nameof(enumType));
-
-            if (!enumType.IsEnum)
-                throw new ArgumentException(SR.Arg_MustBeEnum, nameof(enumType));
+            RuntimeType rtType = ValidateRuntimeType(enumType);
```

Comment:
Technically there's a breaking change here, in that previously if the enum type was not a runtime type and either value was null or format was null, we'd have thrown an ArgumentNullException and now we'll do the RuntimeType check first and throw an ArgumentException.  But it's still misuse either way, and the chances that you have an enum type that's not a RuntimeType and are also passing in a null value or format and that you're catching the ArgumentNullException and thus care seems... remote.